### PR TITLE
Add Alacritty Terminal

### LINKED ISSUE #1115: Alacritty - OpenGL terminal emulator
Project homepage: https://github.com/alacritty/alacritty

Can this app run on RPi? If so, please provide a link to a tutorial if applicable.

- Yes. I was only able to compile it for armhf though.

Is this app something many GUI users would find useful?

- Yes, it's a terminal emulator which uses the GPU for rendering.

Is this app installable with `sudo apt install`?

- No, it's not available in the rpi repositories.

If you've created a zip file for this app, upload it here.

- No zip file at the moment, but it's on my list.
- I have a .deb file in my repository, can be downloaded here: https://apt.raspbian-addons.org/debian/pool/alacritty_0.6.0_armhf.deb

### PR DESCRIPTION
**Pull request by ryanfortner**
to close #1115

### CODE REVIEW

**Comment by ryanfortner** [comment]
@Botspot ready for merge?

**Comment by cycool29** [comment]
The deb seems outdated, I have compiled the deb with the latest Alacritty, tested works well on armhf. 

armhf: https://transfer.sh/D0vbQ0/alacritty_0.9.0_armhf.deb
arm64: Still working...

**Comment by ryanfortner** [comment]
I will test your deb as soon as I can and push it to my repo.

File: apps/Alacritty/install-32 [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+#wget https://apt.raspbian-addons.org/debian/pool/alacritty_0.6.0_armhf.deb || error 'Failed to download alacritty_0.6.0_armhf.deb!'
+#"${DIRECTORY}/pkg-install" "$HOME/alacritty_0.6.0_armhf.deb" "$(dirname "$0")" || error 'Failed to install "alacritty_0.6.0_armhf.deb" with pkg-install!'
+install_packages "https://apt.raspbian-addons.org/debian/pool/alacritty_0.9.0_armhf.deb"
+sudo ln -s /opt/alacritty/alacritty /usr/local/bin/alacritty || error 'Failed to create symbolic link from "/opt/alacritty/alacritty" to "/usr/local/bin/alacritty"!'
```
Comment (cycool29) [inline apps/Alacritty/install-32:13]: This line is not necessary now since the binary is already in `/usr/bin`.

File: apps/Alacritty/install-64 [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+install_packages "https://apt.raspbian-addons.org/debian/pool/alacritty_0.9.0_arm64.deb"
+sudo ln -s /opt/alacritty/alacritty /usr/local/bin/alacritty || error 'Failed to create symbolic link from "/opt/alacritty/alacritty" to "/usr/local/bin/alacritty"!'
```
Comment (cycool29) [inline apps/Alacritty/install-64:11]: Same.

**Pull request merged**